featurebyte/query_graph/sql/interpreter/preview.py

```diff
@@ -998,64 +1021,63 @@ def construct_describe_queries(
     def construct_value_counts_sql(
         self,
         node_name: str,
+        column_names: list[str],
         num_rows: int,
         num_categories_limit: int,
         seed: int = 1234,
-        convert_keys_to_string: bool = True,
```

Comment:
Just to double check it is intentional to remove this key.

Reply:
Yes, it is intended. The flag was added for transitioning to the new behaviour (not converting keys to string), and all callers of this function have already transitioned to setting this flag to False.